qiita_db/test/test_base.py

```diff
@@ -83,7 +85,7 @@ def setUp(self):
 
     def test_status(self):
         """Correctly returns the status of the object"""
-        self.assertEqual(self.tester.status, "in_construction")
+        self.assertEqual(self.tester.status(), "in_construction")
 
```

Comment:
This also needs inside transaction checks for `_status_setter_checks`, `check_status`, and all other functions that are transactified.